static/scss/answers/cards/document-standard.scss

```diff
@@ -64,6 +64,7 @@
   &-detailsText
   {
     @include rich_text_formatting;
+    color: var(--yxt-color-text-primary);
```

Comment:
is it possible to just define text color at top level for as `--yxt-color-text-primary` and let other element by overriding it with their own color like --yxt-color-text-primary? So this doesn't have to be included in every classname related to text

Reply:
We could, however I discussed that with Tom and we decided to be more specific when placing the color because there is less risk of side effects

Reply:
i see. sounds good!